daemon/src/main/scala/io/gearpump/cluster/worker/Worker.scala

```diff
@@ -361,12 +363,19 @@ private[cluster] object Worker {
         // Falls back to workerConfig
         .withFallback(workerConfig)
 
+      // Minimum supported akka.scheduler.tick-duration on Windows is 10ms
+      val duration = config.getInt(Constants.AKKA_SCHEDULER_TICK_DURATION)
+      val updatedConf = if (akka.util.Helpers.isWindows && duration < 10) {
+        LOG.warn(s"akka.scheduler.tick-duration on Windows must be larger than 10ms, set to 10ms")
+        config.withValue(Constants.AKKA_SCHEDULER_TICK_DURATION, ConfigValueFactory.fromAnyRef(10))
+      } else {
```

Comment:
do we check for other invalid duration values (e.g. -1) somewhere ? Is there default value ?